tests/unit/plugins/module_utils/test_config_loader.py

```
@@ -0,0 +1,102 @@
+from __future__ import (absolute_import, division, print_function)
```

Ruff (0.16.1)

[warning] Unnecessary `__future__` imports `absolute_import`, `division`, `print_function` for target Python version

Remove unnecessary `__future__` import

(UP010)